modules/ecs-service/main.tf

```diff
@@ -201,7 +201,7 @@ locals {
   }
 
   portMappings = !var.create_scheduled_task && var.enable_service_connect ? [{
-    name          = random_id.service_connect[0].hex
+    name          = sha256(var.ecs_service_name)
```

Comment:
You're encrypting the ecs service name?

Reply:
Hashing just to make a unique id that isn't tied to another resource lifecycle